configs/sst_networking-firewall-tools.yaml

```diff
@@ -10,17 +10,17 @@ data:
   - c9s
 
   packages:
-  - arptables
-  - arptables-services
   - conntrack-tools
-  - ebtables
-  - ebtables-services
   - firewall-applet
   - firewall-config
   - firewalld
   - ipset
+  - ipset-libs
   - ipset-service
-  - iptables
```

Comment:
I don't think this needs to be removed. I'm sure other SST workloads pull in `iptables`. IIRC, `iptables-nft` has a `Provides: iptables`. That being said, it's better to be explicit with the variant so I think this removal can stay a removal.